baseapp-auth/baseapp_auth/rest_framework/jwt/serializers.py

```diff
@@ -26,13 +26,12 @@ def get_claim_serializer_class(cls):
 class BaseJwtLoginSerializer(
     CustomClaimSerializerMixin, LoginPasswordExpirationMixin, TokenObtainPairSerializer
 ):
-    @classmethod
-    def get_token(cls, user):
+    def get_token(self, user):
```

Comment:
Why did we remove the classmethod decorator here?

Reply:
he is passing `self.context` to the serializer, that wouldn't be possible with a class method

Reply:
Reverted